versioned_docs/version-v1.28.x/getting-started/release-notes/v1_28_0.md

```diff
@@ -53,12 +56,22 @@ The following enhancements were added to the **z/OS FTP Plug-in for Zowe CLI**:
 ### Zowe Application Framework
 
 
-
 ### Zowe CLI
 
 The following bugs were fixed in **core Zowe CLI**:
 
+- Provided more accurate output for `zowe zos-jobs delete job` and `zowe zos-jobs cancel job` commands ([#1333](https://github.com/zowe/zowe-cli/issues/1333)).
+- Fixed inconsistent case on the `modify-version` option for `zowe zos-jobs delete job` and `zowe zos-jobs cancel job` commands ([#1333](https://github.com/zowe/zowe-cli/issues/1333)).
+- Fixed issue where SSH command waits forever when user has an expired password ([#989](https://github.com/zowe/zowe-cli/issues/989)).
+
 The following bugs were fixed in the **Imperative CLI Framework**:
 
+- Updated Imperative to consume security updates in `4.18.2`.
```

Comment:
Not sure if this line is needed, we updated Imperative to consume a security update in the `moment` dependency, but the next line (removing the `moment` dependency) supersedes this.